### DISCUSSION TITLE
missing files in pip install eopsin-lang

### DISCUSSION THREAD

**Discussion by reitcircles** [OpShin/opshin#31] [Q&A] [answered]
The entire directory optimize is missing in pip install. As a result getting error during eopsin build. Just tried copying the optimize directory(and making it a module using __init__.py file inside optimize directory) and it works seamlessly as mentioned in the instructions of https://pycardano.readthedocs.io/en/latest/guides/plutus.html

**Comment by nielstron** [comment, marked as answer]
Thanks, I have fixed this in 709283ccbf687eb57329bae65f6c9493fc75975a

**Comment by nielstron** [comment]
Closing this as it should be solved since #28
